packages/main/src/plugin/mcp/mcp-registry.ts

```diff
@@ -549,6 +550,22 @@ export class MCPRegistry {
     return data;
   }
 
+  registerInternalMCPServer(server: MCPServerDetail): void {
```

Comment:
suggestion(non-blocking): as the class is quite long (> 600 lines), can we add these methods in another class and expose an instance publicly here?